backend/danswer/background/update.py

```diff
@@ -77,7 +77,7 @@ async def run_update():
         try:
             # TODO (chris): make all connectors async + spawn processes to
             # parallelize / take advantage of multiple cores + implement retries
-            document_ids: list[str] = []
+            document_ids = []
```

Comment:
mypy doesn't like this so I removed it. tldr of error: dont declare types if not at the top of the function or something like that

Reply:
are you sure? That doesn't seem right 🤔 maybe it's because it doesn't think this is a typed function since it's missing type hints? Can we change it to `run_update() -> None:` and see if it likes that

Reply:
ah yes, true, you're right, it said so in the error hehe